worth2/templates/goals/goal_setting_block.html

```diff
@@ -1,14 +1,14 @@
 {% load bootstrap %}
 {% load avatar %}
 
-<div class="goal-setting">
+<div class="goal-setting" id="goal-setting-block">
```

Comment:
Correct. Yeah the way I'm working with pagetree for these forms is definitely designed around having just a single customized pageblock on the page.